setup/db/db/schema-480to481-cleanup.sql

```diff
@@ -18,3 +18,6 @@
 --;
 -- Schema cleanup from 4.8.0 to 4.8.1;
 --;
+
+DELETE FROM `cloud`.`host_details` where name = 'vmName' and  value in (select name from `cloud`.`vm_instance`  where state = 'Expunging' and hypervisor_type ='BareMetal');
```

Comment:
Why is this change scoped only to the baremetal hypervisor?  It would seem that it should apply to all hypervisors.